lib/web/desktop/playback.go

```diff
@@ -51,7 +51,7 @@ type Player struct {
 // a particular session.
 type Streamer interface {
 	// StreamSessionEvents streams all events from a given session recording. An error is returned on the first
-	// channel if one is encountered. Otherwise it is simply closed when the stream ends.
+	// channel if one is encountered. Otherwise the event channel is closed when the stream ends.
```

Comment:
So many duplicate docs too  :(

Reply:
Don't worry Alan, I have a nice list of lib/events cleanup and dead code removal that I just didn't get to in time for Teleport 9. We'll get it cleaned up!

Reply:
Nice to hear that :)